csharp/AppEncryption/Directory.Build.props

```
@@ -1,5 +1,5 @@
 <Project>
   <PropertyGroup>
-    <Version>0.2.0</Version>
+    <Version>0.3.0</Version>
```

moving forward, dependency updates will be treated as patch releases
```suggestion
    <Version>0.2.1</Version>
```